pkg/kube_config_manager/kube_config_manager.go

```diff
@@ -108,6 +117,24 @@ func (kcm *KubeConfigManager) KubeConfigEventCh() chan config.KubeConfigEvent {
 	return kcm.configEventCh
 }
 
+// UpdateModuleConfig updates a single module config
+func (kcm *KubeConfigManager) UpdateModuleConfig(moduleName string) error {
+	newConfig, err := kcm.backend.LoadConfig(kcm.ctx)
```

Comment:
I think we have to add variadic parameter `moduleName ...string` to the LoadConfig function, to have ability to reload only desired ModuleConfig

Reply:
done